0-bootstrap/terraform.tfvars

```diff
@@ -0,0 +1,27 @@
+/**
```

Comment:
Looks like this was accidentally committed?

Reply:
Spot on.  Let me remove that.  I was going to submit a future PR modifying the .gitignore file a bit and haven't done that yet.